Core R96 - partially address #120 (#127)
eea-oasis/baseline-standard: core/baseline-core-v1.0-psd01.md

A legal contract that defines the general terms and conditions governing the entire scope of products commercially exchanged between the parties to the agreement.

> **Material impact**

Author: ```suggestion
**Material Impact**
```

BPI service orchestration utilized in a BPI MUST isolate between data source and data consumers.

#### **[R96]** 

> BPI service orchestration MUST not materially impact the overall system latency of the BPI.

Author: ```suggestion
BPI service orchestration MUST not have a Materially Impact on the overall system latency of the BPI.
```

> BPI service orchestration MUST not materially impact the overall system latency of the BPI.

Author: ```suggestion
BPI service orchestration MUST not have a Material Impact on the overall system latency of the BPI.
```
